coalib/results/Diff.py

```diff
@@ -244,10 +245,9 @@ def original(self):
         """
         Retrieves the original file.
         """
-        return self._file
+        return self._original
 
-    @property
-    def modified(self):
+    def _raw_modified(self):
```

Comment:
Hmm... and now I'm lost on what way to take :D
Let's take yours then, it's definitely a valid approach and your work is then not thrown away just by changing the design ;)

btw: `s/originally/original` (damn sometimes I do some grammatical mistakes... that comes from always changing a sentence :3)